composer/loggers/mlflow_logger.py

```diff
@@ -96,13 +98,15 @@ def __init__(
         self.synchronous = synchronous
         self.log_system_metrics = log_system_metrics
         self.ignore_metrics = [] if ignore_metrics is None else ignore_metrics
-        self.ignore_hyperparameters = [] if ignore_hyperparameters is None else ignore_hyperparameters
+        self.ignore_hyperparameters = ([] if ignore_hyperparameters is None else ignore_hyperparameters)
         if self.model_registry_uri == 'databricks-uc':
             if len(self.model_registry_prefix.split('.')) != 2:
                 raise ValueError(
                     f'When registering to Unity Catalog, model_registry_prefix must be in the format ' +
                     f'{{catalog_name}}.{{schema_name}}, but got {self.model_registry_prefix}',
                 )
+        self.run_group = run_group
```

Comment:
same minor nit: nice to have next to `self.run_name`

Reply:
sure, done!